crates/geyser-consumer/src/handler/instructions/sell.rs

```diff
@@ -119,29 +119,30 @@ pub async fn upsert_into_listings_table<'a>(client: &Client, row: Listing<'stati
         .run(move |db| {
             let auction_house: Pubkey = row.auction_house.to_string().parse()?;
 
-            let listing_exists = select(exists(
-                listings::table.filter(
+            let indexed_listing_slot: Option<i64> = listings::table
+                .filter(
                     listings::trade_state
                         .eq(row.trade_state.clone())
                         .and(listings::metadata.eq(row.metadata.clone())),
-                ),
-            ))
-            .get_result::<bool>(db)?;
+                )
+                .select(listings::slot)
+                .first(db)
+                .optional()?;
 
             let listing_id = mutations::listing::insert(db, &row)?;
 
-            if listing_exists {
+            if Some(row.slot) == indexed_listing_slot
```

Comment:
What is the case where we start a a backfill and the version of the listing is older than the 1 we have saved?

Reply:
The slot_wv trigger would not update the listing but we record the activity